packages/core/src/_common/utility.js

```diff
@@ -95,6 +95,23 @@ const isSyntheticsUnavailable = country => countries_without_synthetics.includes
 const blocked_options_countries = ['au'];
 const isOptionsBlocked = country => blocked_options_countries.includes(country);
 
+// Fuction to direct user based on the selected platform from deriv-com before login
+const directUser = platform => {
+    const dmt5_url = 'mt5';
+    const dbot_url = 'bot';
+    const derivx_url = 'derivx';
+
+    if (platform === dmt5_url) {
+        window.location = `/${dmt5_url}`;
+    } else if (platform === dbot_url) {
+        window.location = `/${dbot_url}`;
+    } else if (platform === derivx_url) {
+        window.location = `/${derivx_url}`;
+    } else if (platform !== dmt5_url && platform !== dbot_url && platform !== derivx_url && platform !== null) {
+        window.location = `/`;
+    }
```

Comment:
```suggestion
const directUser = platform => {
    const supported_platforms = ['mt5', 'bot', 'derivx'];
    
    if (!supported_platforms.includes(platform)) {
        window.location = '/';
    }
    window.location = `/${supported_platforms[platform]}`
```

Reply:
Yeah, this will only need to add the condition that states that it will direct him/her to '/' also if the platform !== null, otherwise it will become infinity!

But cool short codes!

Reply:
if the value of `platform` is null or undefined, includes will return false. Hence, the null check is actually handled in the includes line